index.html

```diff
@@ -1,21 +1,22 @@
 <!--
-    Copyright (c) 2020, RTE (http://www.rte-france.com)
+    Copyright (c) 2024, RTE (http://www.rte-france.com)
     This Source Code Form is subject to the terms of the Mozilla Public
     License, v. 2.0. If a copy of the MPL was not distributed with this
     file, You can obtain one at http://mozilla.org/MPL/2.0/.
 -->
+
 <!DOCTYPE html>
 <html class="singlestretch-parent" lang="en" translate="no">
     <head>
-        <base href="%PUBLIC_URL%/" />
+        <base href="/" />
```

Comment:
would it cause problem when use with a sub-path like on azure demo ?  like `demo.gridsuite.com/gridstudy/`

Reply:
Forgot to change a sed in the Dockerfile, thanks